components/selection/selection-observer-mixin.js

```diff
@@ -7,7 +7,12 @@ export const SelectionObserverMixin = superclass => class extends superclass {
 			/**
 			 * The selection info (set by the selection component).
 			 */
-			selectionInfo: { type: Object }
+			selectionInfo: { type: Object },
+			/**
+			 * Id of the collection this component wants to subscribe to (if not located with that collection)
+			 */
+			subscribedTo: { type: String, reflect: true, attribute: 'subscribed-to' },
```

Comment:
I don't feel strongly either way on the naming of this property.  If we wanted to hide the "subscribe" implementation detail from the naming, we _could_ opt for something like `for`.  🤔

Reply:
Since it's a mixin I worry more about name collisions... haha I'd almost go the other way and call it `selection-subscribed-to` or `selection-for`.

Reply:
Seems like `selection-for` handles both concerns (doesn't give away implementation details, and is a bit more unique)